<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrateWidget</class>
 <widget class="QWidget" name="CalibrateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>0</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 35px; color: black;</string>
   </property>
   <property name="text">
    <string>Gauge Diameter:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="diameter_input">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 35px; color: black;</string>
   </property>
   <property name="text">
    <string>Threshold Value:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="threshold_input">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>110</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_image_file">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Select Image File</string>
   </property>
  </widget>
  <widget class="QLabel" name="original_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>930</width>
     <height>510</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: brown;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px; color: black;</string>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="binarized_image">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>330</y>
     <width>930</width>
     <height>510</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: brown;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1430</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px; color: black;</string>
   </property>
   <property name="text">
    <string>Binarized</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preview_btn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>200</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calibrate_btn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>240</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </widget>
  <widget class="QLabel" name="calibration_factor_label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>840</y>
     <width>881</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 35px; color: lightgreen; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Calibration Factor: </string>
   </property>
  </widget>
  <widget class="QComboBox" name="diameter_unit_selection">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
